appengine/endpoints/README.md

```diff
@@ -40,7 +42,7 @@ Open the `swagger.yaml` file and in the `host` property, replace
 
 Then, deploy the sample using `gcloud`:
 
-    gcloud app deploy app.yaml
+    gcloud beta app deploy
```

Comment:
Nice!  Should this be `gcloud app deploy`, though?

Correct to remove app.yaml, but deploying Google App Engine flexible environment applications isn't under `beta` [[1]](https://cloud.google.com/sdk/gcloud/reference/app/deploy)